 Successfully sent 1 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
 Successfully sent 0 reminder(s) for tomorrow's appointments.
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
^C^C^C^C^CPHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
^C^C^C^C^C^C^C^C^CPHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
^C^C^C^C^C^C^C^C^C^C^CPHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
^CPHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
^CPHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
PHP Fatal error:  Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15

Fatal error: Uncaught mysqli_sql_exception: Nem hozható létre kapcsolat, mert a célszámítógép már visszautasította a kapcsolatot in C:\XAMPP\htdocs\project\src\api\config.php:15
Stack trace:
#0 C:\XAMPP\htdocs\project\src\api\config.php(15): mysqli_connect('localhost', 'root', Object(SensitiveParameterValue), 'brickelacuts')
#1 C:\XAMPP\htdocs\project\src\api\cron\send-reminders.php(2): require_once('C:\\XAMPP\\htdocs...')
#2 {main}
  thrown in C:\XAMPP\htdocs\project\src\api\config.php on line 15
^C